transport-native-io_uring/src/main/java/io/netty/channel/uring/IOUringCqe.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2020 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.channel.uring;
+
+final class IOUringCqe {
```

Comment:
Perhaps drop the IOUring prefix? It makes the code more verbose to read

Reply:
yeah uring is already in the package and it's not shared.

Reply:
We can rename package `io.netty.channel.uring` to `io.netty.channel.iouring` and drop all IOUring prefix for internal classes.

Reply:
@hyperxpro good idea, let's do that 👍 what about public classes? :)

Reply:
We can keep `IOUring` prefix in public classes such as `IOUringEventLoopGroup`.